app/controllers/applications_controller.rb

```diff
@@ -10,6 +10,9 @@ def create
 
   def show
     @application = Application.find(params[:id])
+    if params[:name]
+      @pets = Pet.where("name LIKE ?", params[:name])
+    end
```

Comment:
I think we want to avoid using any AR methods other than .all and .find in our controller, so a method like this should typically live within the model. 

I was notice that we have a custom method in the Active Record Model for self.search that you can leverage for this case (so no need to make a new one - yay!). I think it would look something like Pet.search(params[:name]) since :name is the variable used in the application show page search form.

Other than that, it looks fantastic!

Reply:
awesome!! my most recent PR has this change request completed! as we discussed in slack chat, the `.search` AR method utilized instead should be a-okay within the controller. data changing logic should absolutely live in the model, but in this case, I think we are good. let me know your thoughts! 😃

Reply:
I see it is fixed in the next PR for user story 5. Approving and merging this one :)